docs/source/pkg.rst

```diff
@@ -38,7 +38,7 @@ An example call to install a few packages on 2 hosts (with hostnames like
 
 .. _pkg-remove:
 
-``--remove``
+--remove
```

Comment:
Is it just my browser where the dark gray on top of light gray looks terrible?

http://ceph.com/ceph-deploy/docs/pkg.html#remove

Any place that is a 2nd level header has a light/medium gray background.  The backticks make it then have an additional box around it that is a darker color.  They just don't seem necessary to me at all.

Here is what's there now:

![screen shot 2015-03-19 at 5 24 27 pm](https://cloud.githubusercontent.com/assets/1627176/6741314/d92e0de8-ce5c-11e4-84af-b55e45a631ed.png)

On my local build after the changes, this becomes:

![screen shot 2015-03-19 at 5 25 05 pm](https://cloud.githubusercontent.com/assets/1627176/6741325/e8f9e832-ce5c-11e4-9927-0745b7b0e7fb.png)

I think it is better.  But maybe it is personal taste.